spec/lib/waterdrop/testing_spec.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+# Nothing really here. Just a stub for Coditsu
```

Comment:
```ruby
RSpec.describe_current do
  pending
end
```

Reply:
@nijikon I copied that file from `spec/lib/waterdrop/contracts_spec.rb` should that one be changed too?

Reply:
Ah, fair. If it is like that in this repository, then we are good.